.github/workflows/release-build.yml

```diff
@@ -101,16 +101,4 @@ jobs:
           upload_url: https://uploads.github.com/repos/kedacore/keda/releases/${{ steps.get-release-info.outputs.id }}/assets?name=keda-${{ steps.get_version.outputs.VERSION }}.yaml
           asset_path: keda-${{ steps.get_version.outputs.VERSION }}.yaml
           asset_name: keda-${{ steps.get_version.outputs.VERSION }}.yaml
-          asset_content_type: application/x-yaml
-
-      # Upload core deployment YAML file to GitHub release (TO BE DELETED FOR v2.12)
-      - name: Upload Deployment YAML file
-        id: upload-core-deployment-yaml
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: https://uploads.github.com/repos/kedacore/keda/releases/${{ steps.get-release-info.outputs.id }}/assets?name=keda-${{ steps.get_version.outputs.VERSION }}-core.yaml
-          asset_path: keda-${{ steps.get_version.outputs.VERSION }}-core.yaml
-          asset_name: keda-${{ steps.get_version.outputs.VERSION }}-core.yaml
-          asset_content_type: application/x-yaml
```

Comment:
This is an error from ur side, webhooks will continue as optional feature, so we shouldn't remove this code (it publishes the yamls without them)

Reply:
Added back this